pkg/configuration/configuration.go

```diff
@@ -54,6 +54,12 @@ const (
 	// defaultCheRequired is the default value for che.required param
 	defaultCheRequired = false
 
+	// varCheUserDeletionEnabled is a way to turn the Che user deletion logic on/off
+	varCheUserDeletionEnabled = "che.user.deletion.enabled"
```

Comment:
I missed when was doing review for the che configuration previously but due to viper specifics in other places we usually use `.` to separate parent properties from child properties. And `_` when we need to define a multi word property name.
So, if we load the configuration form yaml, json, property file into viper it has opportunity to distinguish parent/children properties.
For example:
```suggestion
	varCheUserDeletionEnabled = "che.user_deletion_enabled"
```
could be loaded as
```
che:
  user_deletion_enabled: true
```
vs `varCheUserDeletionEnabled = "che.user.deletion.enabled"` as:
```
che:
  user:
    deletion:
      enabled: true
```
But not sure if it's worth changing it now since the other configuration properties here do not follow that. So, I'm OK to keep it as is for the consistency sake.

Reply:
Ah ok, I didn't know about that. Makes sense though. I'll do that.